air/src/options.rs

```diff
@@ -206,6 +259,14 @@ impl ProofOptions {
         let remainder_max_degree = self.fri_remainder_max_degree as usize;
         FriOptions::new(self.blowup_factor(), folding_factor, remainder_max_degree)
     }
+
+    /// Returns the `[PartitionOption]` used in this instance of proof options.
+    pub fn get_partition_option(&self) -> PartitionOption {
```

Comment:
If we store the options as a single field, this can be renamed to `partition_options()`.